packages/xrpl/src/models/transactions/MPTokenIssuanceCreate.ts

```diff
@@ -112,6 +113,8 @@ export interface MPTokenIssuanceCreateMetadata extends TransactionMetadataBase {
   mpt_issuance_id?: string
 }
 
+/* eslint-disable max-lines-per-function -- Not needed to reduce function */
+
```

Comment:
Is this applied to the entire module or function below?

Reply:
just the function below

Reply:
Ok, move this below the docstring and above the function signature.